### PR TITLE
cctz 2.1 (new formula)

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cctz.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+class Cctz < Formula
+  desc "C++ library for translating between absolute and civil times"
+  homepage "https://github.com/google/cctz"
+  url "https://github.com/google/cctz/archive/v2.1.tar.gz"
+  sha256 "a86b9b2d339f5638a03a4bbcc1d8c632d2c429aecc44f5c548839c239c3b6e38"
+
+  option "with-shared", "Build shared instead of static library"
```
Comment (DomT4) [inline Formula/cctz.rb]: Can this just be built twice and both installed rather than making this an option?

File: Formula/cctz.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  sha256 "a86b9b2d339f5638a03a4bbcc1d8c632d2c429aecc44f5c548839c239c3b6e38"
+
+  def install
+    system "make", "install_hdrs", "PREFIX=#{prefix}"
+    system "make", "install_lib", "PREFIX=#{prefix}"
+    system "make", "install_shared_lib", "PREFIX=#{prefix}"
+  end
+
+  test do
+    (testpath/"test.cc").write <<~EOS
```
Comment (ilovezfs) [inline Formula/cctz.rb]: let's fetch this as a resource

File: Formula/cctz.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        std::tm tm_local;
+      #if defined(_WIN32) || defined(_WIN64)
+        localtime_s(&tm_local, &now);
+      #else
+        localtime_r(&now, &tm_local);
+      #endif
+        std::cout << format("Local: %Y-%m-%d %H:%M:%S\\n", tm_local);
+      }
+    EOS
+    system ENV.cxx, "--std=c++11", "-I#{include}", "-L#{lib}", "-lcctz", "test.cc", "-o", "test"
```
Comment (ilovezfs) [inline Formula/cctz.rb]: `-std=c++11` not `--std=c++11`

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/cctz.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+class Cctz < Formula
+  desc "C++ library for translating between absolute and civil times"
+  homepage "https://github.com/google/cctz"
+  url "https://github.com/google/cctz/archive/v2.1.tar.gz"
+  sha256 "a86b9b2d339f5638a03a4bbcc1d8c632d2c429aecc44f5c548839c239c3b6e38"
+
```
Comment (stephengroat) [inline Formula/cctz.rb]: maybe add a `head` with `head "https://github.com/google/cctz.git"`?

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@alebcay gentle ping on the feedback here

File: Formula/cctz.rb (hunk 4 above)
Comment (ilovezfs) [reply Formula/cctz.rb]: let's not

**Pull request closed without merging**